iOSMoviesApp/Controllers/DiscoveryPage/DiscoveryPageViewController.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  DiscoveryPageViewController.swift
+//  iOSMoviesApp
+//
+//  Created by Tarang Sultania on 08/07/25.
+//
+
+import Foundation
+import UIKit
+import CoreLocation
+
+protocol DiscoveryPageViewControllerDelegate: AnyObject{
```

Comment:
You can inherit DetailsScreenViewControllerDelegate here. 
```suggestion
protocol DiscoveryPageViewControllerDelegate: ViewControllerTableReloadDelegate {
```